Examples/Kinova/SystemIdentification/ExcitingTrajectories/include/EndeffectorConditionNumberOptimizer.h

```diff
@@ -0,0 +1,105 @@
+#ifndef CONDITION_NUMBER_OPTIMIZER_H
```

Comment:
Please remove this since this is supposed to be in sys_id_dev. You should not develop different features in one branch.